docs/code-quality/ca1716.md

```diff
@@ -52,10 +52,20 @@ You can suppress a warning from this rule if you're convinced that the identifie
 
 ## Configurability
 
+### Analyzed types and members surface
+
 If you're running this rule from [FxCop analyzers](install-fxcop-analyzers.md) (and not with legacy analysis), you can configure which parts of your codebase to run this rule on, based on their accessibility. For example, to specify that the rule should run only against the non-public API surface, add the following key-value pair to an .editorconfig file in your project:
 
 ```ini
 dotnet_code_quality.ca1716.api_surface = private, internal
 ```
 
 You can configure this option for just this rule, for all rules, or for all rules in this category (Naming). For more information, see [Configure FxCop analyzers](configure-fxcop-analyzers.md).
+
+### Analyzed symbol kinds
```

Comment:
Please add a small sentence about configurability of analyzed symbol kinds at the end of the `Cause` section.